src/Organizations/OrganizationForm/OrganizationSummaryForm/OrganizationSummaryForm.js

```diff
@@ -84,6 +85,8 @@ function OrganizationSummaryForm({ initialValues }) {
 
   const isEditMode = Boolean(initialValues.id);
 
+  const _organizationTypes = organizationTypes;
```

Comment:
Why do we need another constant with a reference to the "organizationTypes" object?